brainstormforce/astra
# AST-1688 - Offcanvas Shopping cart closed automatically on mobile when scrolled

### Description
Offcanvas Shopping cart when scrolled gets close automatically.

### Screenshots
https://d.pr/v/Yf7eKQ

### Types of changes

Bug fix (non-breaking change which fixes an issue)



### How has this been tested?
**How to reproduce**
1. Add a cart icon to the website
2. Add at least 9-10 products to make it scrollable
3. Check on real mobile phone or browserstack

### Checklist:
